### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by selma-belhadj**
# What does this PR do?
This PR is raised to create a database from a diagram.

## Description of the completed tasks
- [ ] implement the database from the diagram
- [ ] Join tables from many-to-many relationships

## How should it be tested
- clone the repository
`git clone https://github.com/selma-belhadj/Clinic_Database.git`
- navigate to the folder
`cd Clinic_Database`
- checkout the feature
`git checkout db-diagram `

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    FOREIGN KEY (treatment_id) REFERENCES treatments (id)
+);
+
+CREATE TABLE medical_histories_treatments (
+    id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    treatment_id int,
+    medical_history_id INT,
+    FOREIGN KEY (treatment_id) REFERENCES treatments (id),
+    FOREIGN KEY (medical_history_id) REFERENCES medical_histories (id)
+);
```
Comment (julie-ify) [inline schema_based_on_diagram.sql]: - [x] Wonderful job so far, however, kindly create indexes for all FOREIGN KEYS. For example, the index for the `patient_id` foreign key in the `medical_histories` table may look like this; `CREATE INDEX ON medical_histories (patient_id);`.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @selma-belhadj and @Achilles-Dev,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Good code organization✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (Achilles-Dev) [reply schema_based_on_diagram.sql]: Okay, thank you

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @selma-belhadj, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**